dtfs-central-api/src/v1/controllers/utilisation-report-service/get-fee-record-details-by-id.controller/index.ts

```diff
@@ -0,0 +1,39 @@
+import { SessionBank, ReportPeriod } from '@ukef/dtfs2-common';
+import { HttpStatusCode } from 'axios';
+import { Request, Response } from 'express';
+import { ApiError, NotFoundError } from '../../../../errors';
+import { FeeRecordRepo } from '../../../../repositories/fee-record-repo';
+import { mapToFeeRecordDetails } from './helpers';
+
+export type GetFeeRecordDetailsResponseBody = {
+  id: number;
+  bank: SessionBank;
+  reportPeriod: ReportPeriod;
+  facilityId: string;
+  exporter: string;
+};
+
+type GetFeeRecordDetailsByIdResponse = Response<GetFeeRecordDetailsResponseBody | string>;
+
+export const getFeeRecordDetailsById = async (req: Request, res: GetFeeRecordDetailsByIdResponse) => {
```

Comment:
Could we just call this 'getFeeRecordById' to keep the names shorter? (and other similarly named functions and types in this PR?)